components/eamxx/docs/user/model_output.md

```diff
@@ -89,6 +98,12 @@ I/O interface of EAMxx. There are two types of diagnostic outputs:
     - `precip_ice_surf_mass_flux`
     - `precip_total_surf_mass_flux`
     - `surface_upward_latent_heat_flux`
+    - `wind_speed`
```

Comment:
Peter C. was curious if there was a way we could automate this list.  Like have a small script that parses register_diagnostics.cpp for example and records every option...

Maybe too complicated to implement but this is a good place to ask.

Reply:
I think we discussed this on slack: we can certainly produce a script that automates this, but it is not as easy, so I would punt on it for now.

Reply:
To elaborate more, parsing the diagnostics folder is not enough, since some diagnostic classes are able to produce more than one diagnostic (e.g., `WaterPathDiagnostic` can produce Liq/Rain/Rime/Ice/Vap water paths). Also, I don't foresee us adding tens of diagnostics in the future, so I think manual edit of the docs whenever we add one is probably good enough.